packages/client/hmi-client/src/components/operator/tera-operator-inputs.vue

```diff
@@ -2,6 +2,7 @@
 	<ul>
 		<li
 			v-for="(input, index) in inputs"
+			:id="input.id"
 			:key="index"
```

Comment:
Why not use the `input.id` here and get read of the `index`?

Reply:
Sure, will do.